### DISCUSSION TITLE
希望出一个仅Mojang/微软登录或移除Mojang登录

### DISCUSSION THREAD

**Discussion by XiaoQing114514** [Meloong-Git/PCL#1268] [提问] [answered]
为什么只有仅离线登录或仅正版登录，希望出一个仅Mojang/微软的登录，这样看着主页面更和谐。或者直接移除Mojang登录，因为现在Mojang登录几乎已经不支持，而是由Mojang转到微软登录。

**Comment by WForst-Breeze** [comment, marked as answer]
请见 #43 ，在制作完该功能后会移除 Mojang 登录方式~
